src/tests/test_shared.py

```diff
@@ -0,0 +1,130 @@
+    stat,
+    os
+)
+
+try:
+    # Python 3
+    from unittest.mock import patch, PropertyMock
+except ImportError:
+    # Python 2
+    from mock import patch, PropertyMock
+
+
+class TestShared(unittest.TestCase):
+    """Test class for shared.py"""
+
+    @patch("pybitmessage.shared.sqlQuery")
+    def test_isaddress_in_myaddressbook(self, mock_sql_query):
+        """Test if address is in MyAddressbook"""
+        address = "BM-2cUGaEcGz9Zft1SPAo8FJtfzyADTpEgU9U"
+
+        # if address is in MyAddressbook
+        mock_sql_query.return_value = [address]
+        return_val = isAddressInMyAddressBook(address)
+        mock_sql_query.assert_called_once()
+        self.assertTrue(return_val)
+
+        # if address is not in MyAddressbook
+        mock_sql_query.return_value = []
+        return_val = isAddressInMyAddressBook(address)
+        self.assertFalse(return_val)
+        self.assertEqual(mock_sql_query.call_count, 2)
+
+    @patch("pybitmessage.shared.sqlQuery")
+    def test_isaddress_in_mysubscriptionslist(self, mock_sql_query):
+        """Test if address is in MySubscriptionsList"""
+
+        address = "BM-2cUGaEcGz9Zft1SPAo8FJtfzyADTpEe8Sa"
+
+        # if address is in MySubscriptionsList
+        mock_sql_query.return_value = [address]
+        return_val = isAddressInMySubscriptionsList(address)
+        self.assertTrue(return_val)
+
+        # if address is not in MySubscriptionsList
+        mock_sql_query.return_value = []
+        return_val = isAddressInMySubscriptionsList(address)
+        self.assertFalse(return_val)
+        self.assertEqual(mock_sql_query.call_count, 2)
+
+    @patch("pybitmessage.shared.sqlQuery")
+    def test_reloadBroadcastSendersForWhichImWatching(self, mock_sql_query):
+        """Test for reload Broadcast Senders For Which Im Watching"""
+        mock_sql_query.return_value = [
+            ("BM-2cUGaEcGz9Zft1SPAo8FJtfzyADTpEgU9U",),
+        ]
+        # before reload
+        self.assertEqual(len(MyECSubscriptionCryptorObjects), 0)
+
+        # reloading with addressVersionNumber 1
+        reloadBroadcastSendersForWhichImWatching()
+        self.assertGreater(len(MyECSubscriptionCryptorObjects), 0)
+
+    @patch("pybitmessage.shared.os.stat")
+    @patch(
+        "pybitmessage.shared.sys",
+        new_callable=PropertyMock,  # pylint: disable=used-before-assignment
+    )
+    def test_check_sensitive_file_permissions(self, mock_sys, mock_os_stat):
+        """Test to check file permissions"""
+        filename = "path/to/file"
+
+        # test for windows system
+        mock_sys.platform = "win32"
+        result = checkSensitiveFilePermissions(filename)
+        self.assertTrue(result)
+
+        # test for freebsd system
+        mock_sys.platform = "freebsd7"
+        mock_os_stat.return_value = os.stat_result(
+            sequence=[0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+            dict={
+                "st_mode": stat.S_IRUSR,
+                "st_ino": 753,
```

Comment:
use constants

Reply:
os.stat returns stat_result type object, so, while mocking return value stat_obj is required.

Reply:
I mean the numbers should be constants